app/containers/App/index.js

```diff
@@ -40,7 +40,7 @@ export function App({ location }) {
               <Route
                 exact={routeConfig[routeKey].exact}
                 key={index}
-                path={`${location.pathname}${routeConfig[routeKey].route}`}
+                path={`${routeConfig[routeKey].route}`}
```

Comment:
this will break the production build

Reply:
`const rootPath = process.env.NODE_ENV === 'development' ? '' : location.pathname;`
check added